glass/lensing.py

```diff
@@ -61,27 +61,29 @@ def from_convergence(  # noqa: PLR0913
     deflection potential, deflection, and shear maps. The maps are
     computed via spherical harmonic transforms.
 
-    Returns the maps of:
-
-    * deflection potential if ``potential`` is true.
-    * potential (complex) if ``deflection`` is true.
-    * shear (complex) if ``shear`` is true.
-
     Parameters
     ----------
-    kappa:
+    kappa
         HEALPix map of the convergence field.
-    lmax:
+    lmax
         Maximum angular mode number to use in the transform.
-    potential:
+    potential
         Which lensing maps to return.
-    deflection:
+    deflection
         Which lensing maps to return.
-    shear:
+    shear
         Which lensing maps to return.
-    discretized:
+    discretized
         Correct the pixel window function in output maps.
 
+    Returns
+    -------
+        The maps of:
+
+        * deflection potential if ``potential`` is true.
+        * potential (complex) if ``deflection`` is true.
+        * shear (complex) if ``shear`` is true.
```

Comment:
Maybe requires better indentation?

<img width="353" alt="image" src="https://github.com/user-attachments/assets/ec6c4a62-e287-42d3-b397-87144002cf9f">

```suggestion
        The maps of:

            * deflection potential if ``potential`` is true.
            * potential (complex) if ``deflection`` is true.
            * shear (complex) if ``shear`` is true.
```

Reply:
Good idea